code/game/machinery/doors/airlock_types.dm

```diff
@@ -6,7 +6,7 @@
 	icon = 'icons/obj/doors/airlocks/station/command.dmi'
 	assemblytype = /obj/structure/door_assembly/door_assembly_com
 	normal_integrity = 450
-	security_level = 6
+	security_level = AIRLOCK_SECURITY_PLASTEEL_O_S
```

Comment:
Does this and `/obj/machinery/door/airlock/command/glass` have it set to `AIRLOCK_SECURITY_PLASTEEL_O_S` and not `AIRLOCK_SECURITY_PLASTEEL` on purpose?

Reply:
Yea, because it is set to level 6 which gives it the plate that requires insulated gloves to hack (although im not a fan of the unhackability of them)